docs/index.md

```diff
@@ -32,3 +32,4 @@
 ## Other
 
 * [Chat commands](commands.md)
+* [Occ commands](occ.md)
```

Comment:
You need to add it here as well:
https://github.com/nextcloud/spreed/blob/master/mkdocs.yml#L37

```
INFO     -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
              - occ.md
```